musicbot/bot.py

```diff
@@ -1960,7 +1960,7 @@ def check(m):
         self.config.auto_playlist = not self.config.auto_playlist
         await self.safe_send_message(channel, "The autoplaylist is now " + ['disabled', 'enabled'][self.config.auto_playlist])
         if not player.playlist.entries and not player.current_entry and self.config.auto_playlist: #if nothing is queued, start a song
-            song_url = random.choice(player.autoplaylist) 
+            song_url = random.choice(self.autoplaylist) 
             await player.playlist.add_entry(song_url, channel=None, author=None)							
```

Comment:
Rather than repeating what is already in the `on_player_finished_playing` function, this whole section (line 1962-1964) could be simplified even further by:

```py
if self.config.auto_playlist:
    await self.on_player_finished_playing(player)
```